### PR TITLE
doc: add project specific allocation option/process

### PR DESCRIPTION
**Pull request by mhdawson**
As requested in the discussion related to moving the travel
fund to the OpenJS level.

### CODE REVIEW

**Comment by mhdawson** [comment]
@jasnell let me know if this is what you had in mind.

File: MEMBER_TRAVEL_FUND.md [hunk 1]
```diff
@@ -94,6 +94,32 @@ As examples, the purpose of prior sponsored travels includes attending TC39 meet
 attending [OpenJS collaboration summits](https://github.com/openjs-foundation/summit),
 and mentoring [Code and Learn](https://github.com/nodejs/code-and-learn).
 
+## Event/Project specific requests
+
+While members from any OpenJS project can request approval for travel through
+the process outlined in the `Approval` section, there may be situations in which 
+a project wishes to manage the approvals an event or purpose instead of
```
Comment (molant) [inline MEMBER_TRAVEL_FUND.md]: English is not my first language so I might be off, but I don't fully understand this phrase.

> to manage the approvals an event or

Comment (mhdawson) [reply MEMBER_TRAVEL_FUND.md]: Thanks for catching that. Will update.

Comment (mhdawson) [reply MEMBER_TRAVEL_FUND.md]: Fixed.

**Comment by mhdawson** [comment]
I think this falls into the `Governance change` and meets the landing requirements as it has been open >14 days, there are no objections and we have enough approvals.

**Pull request merged**